Tests/Translation/Extractor/File/Fixture/MyFormType.php

```diff
@@ -63,13 +63,6 @@ public function buildForm(FormBuilder $builder, array $options)
                 'label' => false,
                 'attr' => array('placeholder' => /** @Desc("Field with a placeholder but no label") */ 'form.placeholder.text.but.no.label')
             ))
-            ->add('field_with_choice_as_values', 'choice', array(
```

Comment:
[`choices_as_values` is not needed anymore from 3.x](https://github.com/symfony/symfony/blob/3.4/UPGRADE-3.0.md#frameworkbundle)